inc/nux/class-storefront-nux-starter-content.php

```diff
@@ -656,7 +656,7 @@ public function filter_sf_categories( $args ) {
 				$args['hide_empty'] = false;
 
 				// List of categories to display.
-				$args['ids'] = implode( $product_cats, ',' );
+				$args['ids'] = implode( ',', $product_cats );
```

Comment:
TIL that this function [will work with the arguments the wrong (deprecated) way around](https://www.php.net/manual/en/function.implode.php)!

Reply:
It does work, but does that justify running deprecated code? 🤔

Reply:
Of course not, we definitely want the arguments in a non-deprecated order!

Reply:
Than we can mark this part as resolved, right?

Reply:
For sure, I totally approve of this change, was just commenting 😄